internal/backend/fs.go

```diff
@@ -175,6 +209,8 @@ func (s *storageFS) getObject(bucketName, objectName string) (Object, error) {
 	}
 	obj.Name = filepath.ToSlash(objectName)
 	obj.BucketName = bucketName
+	obj.Crc32c = encodedCrc32cChecksum(obj.Content)
+	obj.Md5Hash = encodedMd5Hash(obj.Content)
```

Comment:
I wonder if we should generate those on creation and store in the JSON. What do you think?

Reply:
Disagree, if you start the server then overwrite a file you would have a hash mismatch. You could lock the files but I think it's overkill.

Reply:
One of the points of a checksum is to ensure integrity, and an external process overwriting a file is a break of integrity x) It's a test server though, so no big deal.

Reply:
Oh I merged, but I'm going to revert. The server already behaves accordingly as I suggested, I don't think it's a good idea to have storage backends behave differently on data.

I created an issue for generating the md5 and crc sums when seeding from the disk (see #313).